src/ui/flutter_app/lib/l10n/intl_es.arb

```diff
@@ -753,23 +753,23 @@
     "@takeBack": {
         "description": "Take back"
     },
-    "takingBack": "Retirar...",
+    "takingBack": "Retrocediendo...",
     "@takingBack": {
         "description": "Taking back..."
     },
-    "waiting": "Esperar...",
+    "waiting": "Espere...",
```

Comment:
The latest version of the English translation removes this ellipsis. Not sure if the Spanish custom is to put an apostrophe after it?

Reply:
Similar to previous sentence, it's fine.